app/Main.js

```diff
@@ -25,7 +25,10 @@ try {
 function runApp(initial_state) {
     console.log('Initial state', initial_state);
     const config = initial_state.offchain.config
-    steem.config.set('websocket', config.ws_connection_client);
+    if(process.env.USE_JSONRPC)
```

Comment:
This is duplicating condensers existing functionality with websockets but for http/jsonrpc. As it exists now, condenser makes use of two instances of steem-js's - one for the SPA/client to make (ws_connection_client/http_connection_client) and one for condenser itself. However, these two variables are almost always going to be set to the same URI (as was the case with websockets as well). The USE_JSONRPC env var is used in both places... so either ws_connection_client and ws_connection_server would be set for websockets (as it's always been), or http_connection_client and http_connection_server would be used alternatively if USE_JSONRPC is set. This makes it easy to switch between the two functionalities if necessary (in case a problem is later uncovered with http steem-js for example) and it won't break deployments before the env vars are set/changed.